tap_pendo/streams.py

```diff
@@ -549,20 +550,41 @@ def set_request_body_filters(self, body, start_time, records=None):
 
         return body
 
+    def get_replication_value(self, record):
+        """Return the replication value of """
+        if isinstance(self, Accounts):
+            return record['metadata']['auto']['lastupdated']
```

Comment:
Is there never a top-level lastupdated key on accounts records? If not, we should probably either: 1) remove that field from the schema file, or 2) check with product, but consider raising the metadata field up to the top level in addition to leaving it nested in the metadata_auto object we persist